applications/luci-app-ntpc/htdocs/luci-static/resources/view/ntpc.js

```diff
@@ -0,0 +1,78 @@
+'use strict';
+
+'require form';
+'require rpc';
+'require view';
+
+var callSystemInfo = rpc.declare({
+	object: 'system',
+	method: 'info'
+});
+
+return view.extend({
+	load: function () {
+		return Promise.all([
+			L.resolveDefault(callSystemInfo(), {})
+		]);
+	},
+
+	render: function (data) {
+		var system_info = data[0];
+
+		var m, s1, time, interval, count, freq, s2, s3, hostname, port;
```

Comment:
Usually for section and option separate vars aren't created.

Reply:
Fixed